src/main/java/com/dataloom/data/DataApi.java

```diff
@@ -42,6 +43,13 @@
             @Path( NAME_SPACE ) String entityTypeNamespace,
             @Path( TYPE_NAME ) String entityTypeName );
 
+    @PUT( CONTROLLER + "/" + ENTITY_DATA + "/" + NAME_SPACE_PATH + "/" + TYPE_NAME_PATH + "/" + NAME_PATH + "/" + SELECTED )
```

Comment:
I find that using `TYPE_NAME_PATH` is confusing because it refers to the name property in a `FullyQualifiedName`, and `NAME_PATH` refers to the entity set name. but with other APIs, `NAME_PATH` refers to the name property in a `FullyQualifiedName`.

I'm just sharing my thoughts, so you don't have to do anything. But one idea I have to make this more clear would be to do something like this:

```
CONTROLLER + "/" + ENTITY_DATA + "/" + ET_NAME_SPACE_PATH + "/" + ET_NAME_PATH + "/" + ES_NAME_PATH + "/" + SELECTED
```

... where `ET` is short for `ENTITY_TYPE` and `ES` is short for `ENTITY_SET`. Similarly, you'd have `PT` for `PROPERTY_TYPE`.